scripts/install-multi-user.sh

```diff
@@ -136,7 +136,7 @@ EOF
     cat <<EOF
 $step. Delete the files Nix added to your system:
 
-  sudo rm -rf /etc/nix $NIX_ROOT $ROOT_HOME/.nix-profile $ROOT_HOME/.nix-defexpr $ROOT_HOME/.nix-channels $HOME/.nix-profile $HOME/.nix-defexpr $HOME/.nix-channels
+  sudo rm -rf /etc/nix $NIX_ROOT $ROOT_HOME/.nix-profile $ROOT_HOME/.nix-defexpr $ROOT_HOME/.nix-channels $ROOT_HOME/.local/state/nix $ROOT_HOME/.cache/nix $HOME/.nix-profile $HOME/.nix-defexpr $HOME/.nix-channels $HOME/.local/state/nix $HOME/.cache/nix
```

Comment:
Is there any chance these variables may contain whitespace?
Maybe worth quoting them.